### ISSUE TITLE
Sails model boolean attribute conversion to tinyint in db

### ISSUE THREAD

**Issue by victorbadila** [balderdashy/sails-mysql#218]
This isn't exactly an issue, but I have found out that an attribute in a model which is of type 'boolean' translates in mysql to 'tinyint' type instead of 'boolean'.  Now I know they are very similar, but is there a way to instruct a sails model, or maybe the sails-mysql adapter to map certain waterline types to certain mysql types when building the database from the waterline schema?
The part where this is relevant is the fact that I also have stored procedures running inside mysql and it would be clearer to treat certain attributes as boolean (only true/false values) instead of tinyint.
Cheers!

**Comment by devinivy** [comment]
Sounds right to me!  But it looks like boolean attributes _should_ already do what you're talking about.  Which version of sails-mysql are you using?  https://github.com/balderdashy/sails-mysql/blob/1e7f0b4cf19d790e4f3bab0351335b77b3aeb6c2/lib/sql.js#L378-L380

**Comment by victorbadila** [comment]
0.10.11 and it has the same code as the one you typed there. well, this sure is strange.. :)
Might close this, maybe it's something on my part, although I wouldn't know what.

**Comment by sailsbot** [comment]
Thanks for posting, @victorbadila.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
